### PR TITLE
added comments and order generation

### PR DESCRIPTION
**Pull request by ayushpreenza07**

### CODE REVIEW

File: src/test/java/SalesforceModules/OppurtunitiesUtil.java [hunk 1]
```diff
@@ -166,35 +259,67 @@ public static void verifyDocumentGenerated(String logStep){
...
-        KeywordUtil.click(OppurtunitiesObject.selectClosedStage,logStep);
+        WebElement element2 = KeywordUtil.getDriver().findElement(OppurtunitiesObject.selectClosedStage);
+        executor.executeScript("arguments[0].click();", element2);
 
+        //select closed won value from dropdown
         KeywordUtil.waitForVisible(OppurtunitiesObject.selectClosedStageDropdown);
-        Select dropdown = new Select(KeywordUtil.getDriver().findElement(OppurtunitiesObject.selectClosedStageDropdown));
-        dropdown.selectByValue("Closed Won");
+//        Select dropdown = new Select(KeywordUtil.getDriver().findElement(OppurtunitiesObject.selectClosedStageDropdown));
+//        dropdown.selectByValue("Closed Won");
```
Comment (snaphray) [inline src/test/java/SalesforceModules/OppurtunitiesUtil.java]: Please remove commented lines if not required

File: src/test/java/SalesforceModules/AccountUtil.java [hunk 2]
```diff
@@ -91,17 +101,33 @@ public static void clickSaveButton(String logStep){
 //        executor.executeScript("arguments[0].click();", KeywordUtil.getDriver().findElement(AccountObject.saveButton));
```
Comment (snaphray) [inline src/test/java/SalesforceModules/AccountUtil.java]: Could you please remove this commented line as well if not required?

File: src/test/java/step_definitions/salesforceDemo.java [hunk 3]
```diff
@@ -52,16 +52,16 @@ public void read_the_test_data_from_Excel(String arg1) {
     @When("^navigate to salesforce$")
     public void navigate_to_the_url() throws Exception {
 
-//        KeywordUtil.navigateToUrl("https://testing-f5-dev-ed.develop.my.salesforce.com");
-        dataMap = ExcelDataUtil.getTestDataWithTestCaseID("Salesforce", "TestData1");
-        KeywordUtil.navigateToUrl(dataMap.get("URL"));
+        KeywordUtil.navigateToUrl("https://testing-f5-dev-ed.develop.my.salesforce.com");
+//        dataMap = ExcelDataUtil.getTestDataWithTestCaseID("Salesforce", "TestData1");
```
Comment (snaphray) [inline src/test/java/step_definitions/salesforceDemo.java:56]: Could you please remove these commented lines as well if not required?

**Review by snaphray — CHANGES_REQUESTED** [review]
Please check the comments

File: src/test/java/step_definitions/salesforceDemo.java (hunk 3 above)
Comment (ayushpreenza07) [reply src/test/java/step_definitions/salesforceDemo.java:56]: These are required for now until we find solution for different URLs problem

**Review by snaphray — APPROVED** [review]
Code looks good

**Pull request merged**